packages/geometry/.gitignore

```diff
@@ -0,0 +1 @@
+.lib
```

Comment:
Can prob add `.lib/` to the root `.gitignore` so it applies to packages in the future.